src/current/v22.2/backup-validation.md

```diff
@@ -107,17 +107,17 @@ DROP DATABASE movr CASCADE;
 
 It is important to note that [full cluster restores](restore.html#full-cluster) with `schema_only` will write the system tables to disk. This provides important coverage for validation at the cluster level. Writing the system tables should not have a notable impact on the runtime of this process.
```

Comment:
Hey @msbutler, question re system tables... the issue mentions updating these to "metadata" — do we classify the system tables as metadata? I only ask because we do refer to system tables pretty frequently across the docs.

Reply:
So, I haven't yet changed these instances, but can update as we see best 👍

Reply:
ah, if we refer to system tables elsewhere in the docs, then I'm fine keeping this paragraph as is. Thanks for flagging!